services/src/messaging/src/main/java/org/openkilda/messaging/info/rule/FlowMatchField.java

```diff
@@ -59,5 +62,6 @@ public FlowMatchField(
         this.udpDst = udpDst;
         this.inPort = inPort;
         this.vlanVid = vlanVid;
```

Comment:
Yes, we can. But I would not like to do so, because in the current implementation we will be able to distinguish between `transit_vlan` and `vxlan`. If we make only `tunnel_id`, then it will be unclear what will be written in the discrepancies.

Reply:
It make sense in case of discrepancies. Thanks.